### DISCUSSION TITLE
Update questions

### DISCUSSION THREAD

**Discussion by violenttamper** [Tampermonkey/tampermonkey#1240] [Q&A] [answered]
更新已被删除的脚本，使得脚本失效，请问怎么解决？怎么设置禁止更新某些脚本？

**Comment by Geeyun-JY3** [comment]
用英文啊……

**Comment by violenttamper** [comment]
Update the deleted script to make it invalid. How can I solve this problem? How do I set the disable update of some scripts?

**Comment by Geeyun-JY3** [comment, marked as answer]
猛得发现可以回答你的问题。
第一个问题无解，找备份。下次弹出更新的时候注意别看都不看就点了更新。

第二个问题：点击要关闭更新的脚本标题 > 取消勾选「检查更新」的复选框 > 点击保存
![image](https://user-images.githubusercontent.com/49396421/136654098-0e52fefd-2cd7-4cee-a939-5424aa87ebb1.png)
